component/Card/index.html

```diff
@@ -0,0 +1,264 @@
+        </div>
+        </div>
+    </div>
+    <!-- card with dismiss -->
+    <div class="card">
+      <div class="card-background">
+        <div class="card-body">
+          <h3 class="card-title">Title</h3>
+          <small class="small-text">secondary text</small>
+          <p class="card-description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem quis, expedita ab saepe voluptatum ut ad sequi, quasi dolores enim animi fugit nostrum autem quaerat libero doloremque! Impedit, asperiores dolores.</p>
+          <img class="badge-icon" src="/component/Card/icon/close.png" alt="badge.png">
+        </div>
+      </div>
+    </div>
+    <!-- card with badge -->
+    <div class="card">
+      <div class="card-background">
+        <img
+        class="card-img"
+        src="https://www.planetware.com/wpimages/2020/01/india-in-pictures-beautiful-places-to-photograph-gateway-of-india-mumbai.jpg"
+        alt="img"
+      />
+          <div class="card-body">
+            <h3 class="card-title">Title</h3>
+            <small class="small-text">secondary text</small>
+            <p class="card-description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem quis, </p>
+          </div>
+        <img class="badge-icon" src="/component/Card/icon/like.png" alt="badge.png">
+      </div>
+    </div>
+    <!-- card with shadow -->
+
+    <div class="card ">
+      <div class="card-background shadow">
+        <img
+        class="card-img"
+        src="https://www.planetware.com/wpimages/2020/01/india-in-pictures-beautiful-places-to-photograph-gateway-of-india-mumbai.jpg"
+        alt="img"
+      />
+          <div class="card-body">
+            <h3 class="card-title">Title</h3>
+            <small class="small-text">secondary text</small>
+            <p class="card-description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem quis, </p>
+          </div>
+        <img class="badge-icon" src="/component/Card/icon/share.png" alt="badge.png">
+      </div>
+    </div>
+
+
+
+
+    <!-- basic card -->
+    <div class="card__container">
+      <div class="card__wrapper">
+        <div class="card_background">
+          <img
+          class="image"
+          src="https://www.planetware.com/wpimages/2020/01/india-in-pictures-beautiful-places-to-photograph-gateway-of-india-mumbai.jpg"
+          alt="img"
+        />
+        <div class="card__content">
+          <h3 class="title">Our Changing Planet</h3>
+          <p class="small_content">by Kurt Wagner</p>
+          <p class="description">
+            Visit ten places on our planet that are undergoing the biggest
+            changes today.
+          </p>
+        </div>
+        </div>
+       
+        <footer class="footer">
```

Comment:
Dont use Element `<footer >` in middle of the document. There are some HTML tags that have some semantic meaning.  You can read about the meaning of these elements [here](https://www.w3schools.com/html/html5_semantic_elements.asp)

And don't use the class `footer`.

If I'm not wrong, the bottom part has links and buttons. You can wrap them in a div and give a class name as below

```
<div class="card-links-container">
....content
</div>
```

Reply:
ok